src/System.IO.FileSystem/tests/Enumeration/Win32MatcherTests.netcoreapp.cs

```diff
@@ -112,5 +129,19 @@ public static void Win32Match(string expression, string name, bool ignoreCase, b
             { @"<.", @"a.", true, true },
             { @"<.", @"a.b", true, false },
         };
+
+        [Theory,
+            InlineData("", "*"),
```

Comment:
That's an interesting way of specifying the test data. I didn't know you could do that.

I have almost always seen it as:
```C#
[Theory]
[InlineData("", "*")]
[InlineData("*.*", "*")]
...
public void TranslateExrpression(string expression, string expected)
```